itez/beneficiary/models.py

```diff
@@ -72,3 +73,58 @@ class WorkDetail(models.Model):
 
     def __str__(self):
         return self.company
+
+
+class AgentDetail(models.Model):
+    """
+    create agent detail table with its instances/columns.
+    """
+    MALE        =  1
+    FEMALE      =  2
+    TRANSGENDER =  3
+    OTHER       =  4
+    GENDER_TYPES = (
+        (MALE, "Male"),
+        (FEMALE, "Female"),
+        (TRANSGENDER, "Transgender"),
+        (OTHER, "Other")
+    )
+    role = models.ForeignKey(
+        FieldPersonnel,
+        on_delete=models.CASCADE
+    )
+    first_name = models.CharField(
+        _("First Name"),
+        max_length=200,
+        null=False
+    )
+    second_name = models.CharField(
+        _("Second Name"),
+        max_length=200,
+        null=False
+    )
+    birthdate = models.DateField(
+        _("Birth Date"),
+        auto_now_add=False,
+        null=False
+    )
+    agend_id = models.TextField(
+        _("Agent Id"),
+        null=False,
+        default=generate_uuid_and_agent_code()[1],
+        max_length=100
+    )
+    gender = models.CharField(
+        _("Gender"),
+        max_length=200,
+        choices=GENDER_TYPES,
+        default="Other"
+    )
+    assigned_location = models.CharField(
+        _("Assigned Location"),
+        null=False,
+        max_length=500
+    )
+
+    def __str__(self):
+        return "%s %s" %(self.first_name, self.second_name)
```

Comment:
lets not use this string representation
lets use `f` strings like

`return f"{self.first_name} {self.second_name}"`